Introduction (#29)
monstar-lab-oss/API-manifesto: README.md

## Introduction

> This API Manifesto is a fast & easy overview over the most important elements for building a rock solid API, which both the Backend & Frontend team enjoys working with.

Reviewer: ```suggestion
This API Manifesto is a fast and easy overview of the most important elements for building a rock-solid API, which both the backend and frontend team enjoys working with.
```

> APIs are suppose to be very strict, it's a contract between Backend & Frontend. In the same time, there is no reason for APIs to be different depending on which language / framework was used in the Backend. 

Reviewer: ```suggestion
APIs are supposed to be very strict and to be considered as a contract between backend and frontend. Further, we believe that there is no reason for APIs to be different depending on which language or framework that has been used in the backend. 
```

> This is not a full blown manifesto, check the [Inspiration section](#inspiration) for full content

Reviewer: ```suggestion
This is not a full blown manifesto, check the [Inspiration section](#inspiration) for references to other manifestos with more details.
```
